### PR TITLE
Create consolidated APIView comment

### PR DESCRIPTION
**Pull request by praveenkuttappan**
APIView bot writes a comment even if api changes are not present for a package. This was done as per the feedback but this is causing bot to spam with multiple comments when PR touches large number of packages. Disabling no change comment as interim solution until we come up with a solution to create single bot comment from apiview for entire PR.

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs [hunk 1]
```diff
@@ -162,13 +162,19 @@ private async Task<PullRequestModel> GetPullRequestModel(int prNumber, string re
             if (pullRequestModel == null)
             {
                 string[] repoInfo = repoName.Split("/");
-                var issue = await _githubClient.Issue.Get(repoInfo[0], repoInfo[1], prNumber);
+                var pullRequest = await _githubClient.PullRequest.Get(repoInfo[0], repoInfo[1], prNumber);
+                if(pullRequest.Draft == true)
```
Comment (weshaggard) [inline src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs:166]: How do we catch new APIs when the shift out of draft mode but no CI is triggered?

File: src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs [hunk 2]
```diff
@@ -281,13 +287,8 @@ private ReviewModel CreateNewReview(PullRequestModel prModel)
...
                 if (await IsReviewSame(review, renderedCodeFile))
                 {
-                    //Do not update the comment if review was already created and it matches with current revision.
-                    if (pullRequestModel.ReviewId != null)
-                        return "";
-
-                    //Baseline review was not created earlier or this is the first commit of PR
-                    stringBuilder.Append($"API changes are not detected in this pull request for `{codeFile.PackageName}`");
-                    return stringBuilder.ToString();
+                    // Disable the comment when API change is not detected.
```
Comment (weshaggard) [inline src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs:290]: Do we want to disable the comment? My expectation is that we would just reuse the existing comment if it already exists and if not then we would add one saying there is no updates detected.

Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs:290]: We are already reusing existing comment. But this is still an issue when a PR touches multiple services it gets one comment per each package. We still don't have combined single comment for entire PR. It is one comment per package which can still be an issue. This PR is temporary solution until we build combined single comment taht includes link for each PR.

File: src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs (hunk 1 above)
Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs]: I have reverted this change and updated PR to create consolidated comment for all packages

File: src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs [hunk 3]
```diff
@@ -272,51 +279,24 @@ private ReviewModel CreateNewReview(PullRequestModel prModel)
...
                 if (await IsReviewSame(review, renderedCodeFile))
                 {
-                    //Do not update the comment if review was already created and it matches with current revision.
-                    if (pullRequestModel.ReviewId != null)
-                        return "";
-
-                    //Baseline review was not created earlier or this is the first commit of PR
-                    stringBuilder.Append($"API changes are not detected in this pull request for `{codeFile.PackageName}`");
-                    return stringBuilder.ToString();
+                    return;
```
Comment (weshaggard) [inline src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs:289]: Do we need to store this info so we can add it to the aggregate comment?

Comment (weshaggard) [reply src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs:289]: Can we not add unmodified packages to the single comment and with just a note that there are no changes? Or maybe even a link to the diff like we talked about with SDK automation where it is a link to an API diff that has no changes.

Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs:289]: @weshaggard : I have created an issue for tracking of this change to show unchanged packages list. https://github.com/Azure/azure-sdk-tools/issues/3348 

Can I go ahead and merge this PR?

**Review by weshaggard — APPROVED** [review]

**Pull request merged**